Add env var overrides for controller enable flags (#3181)
DataDog/datadog-operator: docs/control_plane_monitoring.md

helm install datadog-operator datadog/datadog-operator --set introspection.enabled=true
```

> Or, for **OpenShift users** who installed the operator via OperatorHub/Marketplace (the [recommended method](install-openshift.md)), set the environment variable in the `Subscription`:

Reviewer: ```suggestion
Or, for **OpenShift users** who installed the operator using OperatorHub/Marketplace (the [recommended method](install-openshift.md), because it survives operator upgrades), set the environment variable in the `Subscription`:
```
Feel free to correct this phrasing if it's not accurate; I just thought it was worth mentioning sooner that this is the recommended method, in case we can save the user a bit of thinking 🙂

```yaml
apiVersion: operators.coreos.com/v1alpha1
kind: Subscription
metadata:
  name: datadog-operator
spec:
  # ... channel, source, etc.
  config:
    env:
      - name: DD_INTROSPECTION_ENABLED
        value: "true"
```

> Alternatively, you can patch the CSV directly (the env var approach above is preferred as it survives operator upgrades):

Reviewer: ```suggestion
Alternatively, you can patch the CSV directly, which does not persist after operator upgrades:
```
